Procurement/RelayCommand.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Diagnostics;
+using System.Windows.Input;
+
+namespace Procurement.View.ViewModel
+{
+    public class RelayCommand : ICommand
+    {
+        #region Fields
```

Comment:
I'd like to avoid adding regions, doubly so in a small class like this.

Reply:
Agreed.